---
title: "Quantitative, Multispecies Monitoring at a Continental Scale"
author:  
  - Gledis Guri* 
  - Owen Liu*
  - Ryan P. Kelly
  - Megan Shaffer
  - Kim Parsons
  - Krista Nichols
  - Andrew Olaf Shelton
  - Many others, in some order
format: pdf
editor: visual
---

Target: PNAS, end of 2024

```{r}
#| echo: false
suppressMessages(library(here))
suppressMessages(library(tidyverse))

```

## Abstract

Molecular data from environmental samples can reflect the abundance of species' DNA, an index immediately relevant to natural resources management at a continental scale. These data commonly derive from qPCR or ddPCR assays --- which focus on a single species --- or metabarcoding assays from more general PCR primers that amplify many species simultaneously. Multispecies analyses make efficient use of field samples and laboratory time, and speak to inherently multispecies questions of management and ecological research. However, unlike single-species techniques, metabarcoding by itself can reflect only the proportions of target-species DNA present, not their absolute quantities. Here, we combine qPCR and metabarcoding data derived from the same samples to map quantities of many fish species along the US Pacific coast in three dimensions, demonstrating a technique of practical importance for both management and ecology. We derive spatially explicit estimates of eDNA abundance for \[13\] common species of commercial and management importance and point the way to quantitative surveys of wild species using molecules alone. <!--Maybe add slightly on the results of species accumulation or that we detect species ecological niches -->

<!--# What is necessary to mention in intro:  -->

<!--# eDNA quantities = organismal abundance + ε [DONE?]  -->

<!--# eDNA MB != quantities [DONE?]  -->

<!--# Multispecies survey saves time and $ [DONE?]  -->

Trawls have challenges (catchability, expensive) \[Not sure how to treat this one; we can't anger the trawl people; we need to engage w them; tread lightly; OLE\]

<!--# eDNA MB helpful tool due to multi species ability [DONE?]  -->

<!--# Circumvent MB quantification problem with qPCR + MB [DONE?]  -->

## Intro

Multicellular organisms are, by definition, composed of cells, most of which contain DNA. Accordingly, where there are more such cells --- a straightforward measure of organismal abundance --- there is more of the species' DNA. At sufficiently large spatial scales, patterns of species' environmental DNA (eDNA) therefore approximate the patterns of abundance of the species themselves, with deviations due to biophysical processes (such as when DNA is transported by currents), observation variance (such as stochastic detection of rare target molecules in the environment), and process variance (as when an animal differentially sheds DNA as a result of injury or life-history stage) <!--\cite{yates_2022}-->. The latter two of these are sources of statistical noise that may be distinguished analytically from biological signal.

Where the spatial scale of the sampling is appropriate to the scale of the species of interest, single-species eDNA data --- generally derived from qPCR or ddPCR analysis --- routinely track organismal abundance in environmental contexts as varied as aquaria \[CITE Jo\], rivers and streams \[CITE\], estuaries and nearshore habitats \[CITE\], and coastal oceans \[CITE\]. <!--\cite{maes_2023} , plus Shelton et al. 2019--> However, mismatches in the spatial scale of sampling vs. biological process can result in a loss of biological signal and therefore in counter-intuitive observations \[CITE\]. For example, discrete water samples taken at 100km intervals are unlikely to reflect eDNA \[signals\] useful for describing patterns in species that vary over much larger or much smaller scales (e.g., large-bodied pelagic predators or krill, respectively). <!--#[CITE that atlantic fisheries paper that looked at ?tuna?]. --> <!--# this is consistent with sampling strategies in use throughout ecology and natural-resources management; the sampling tool must be appropriate to the job at hand-->

Where quantitative analysis is required, single-species eDNA studies have been much more common than multispecies surveys (i.e., metabarcoding, in which characteristic fragments of DNA from a mix of species in an environmental sample) \[CITE\], despite seemingly greater information content in \[the latter\]. Multispecies analysis reflects information on many species of interest simultaneously, but at the analytical costs of requiring 1) correction for species-specific observation efficiency, and 2) a resulting dataset consisting of a composition, rather than of a set of statistically independent observations. Consequently, metabarcoding data themselves cannot yield information about the abundances of organisms in an environmental sample.

Estimating the abundances of wild species is a routine and general challenge in natural resources management; such assessments are often desirable or required by law \[CITE examples\]. For example, under the Magnuson-Stevens Act of 1976, NOAA Fisheries must routinely survey stocks of commercially exploited fish species. This systematic, science-based accounting --- independent of the commercial fishing industry itself --- has been a revolutionary stride toward sustainable management of the nation's natural resources <!--# CITE the Magnuson-Stevens Act at 40 report; https://www.fisheries.noaa.gov/leadership-message/governance-fish-forty-years-under-magnuson-stevens-act -->. However, the cost and effort required to collect often-imprecise estimates of fish stock biomass are substantial, ranging into the \[\$10s of millions\] per \[monthlong\] research cruise. Improving the information yield from these cruises therefore would be a welcome advance.

Here, we provide quantitative, three-dimensional distribution models for \[13\] commercially important species, derived entirely from residual genetic material (eDNA) in a single set of 347 water samples taken as part of a NOAA Fisheries cruise. To do so, we correct for species-specific differences in observation efficiency (i.e., amplification bias) and link corrected metabarcoding proportions to absolute quantities of eDNA via expansion using a reference species quantified with qPCR.

As implemented in commercial fisheries stock assessments, stock assessors develop a unitless index that is linearly proportional to biomass --- a doubling of the index represents a doubling of biomass --- but with the proportionality constant initially undefined \[defined later or approximated, using some denominator... OLE\]. The eDNA-derived estimates we provide here are well-suited to the same treatment, such that they may fold into stock assessments with little additional effort. Moreover, multispecies stock assessments often require multiple gear types to effectively target various fish species. However, using different gear types requires a substantial increase in sampling effort to estimate gear-specific catchability terms for each target species <!--# \cite{zhang_2020} -->, resulting in a tradeoff of spatial resolution vs. precision in abundance estimates. Metabarcoding avoids this tradeoff by surveying many species with the same "gear" in a format that is easily replicated in space and time.

As agencies begin to implement the new National Aquatic eDNA Strategy -- the first goal of which is to fold eDNA data into federal decisionmaking -- this and other large-scale examples point the way to maximizing the benefit of this information-rich datastream for natural-resources management at a continental scale.

## Methods

In brief, we calibrated metabarcoding data, correcting raw species observations to their initial proportions using a community of known composition <!--# \cite{shelton_2022} -->. We then used the absolute quantity of one species to expand the initial estimated proportions into absolute abundance estimates for each species <!--# \cite{guri_2024} and see also Allan et al. 2023--> in each of \[347\] 2L water samples collected during a 2019 NOAA cruise <!--# See Ramon-Laca et al. 2021; Shelton et al. 2022 -->. The details of each step follow below.

<!--# We jointly estimate 1) the standard curve linking laboratory standards to environmental concentrations of reference species DNA using qPCR, 2) the observation (amplification) efficiency for each focal species in a multispecies (metabarcoding) PCR relative to the reference species, and 3) the abundance of each species' DNA in each environmental sample. A similar analysis featured in Guri et al. (2024). The details of each step follow below. -->

### Sample Collection and Processing

Samples were collected aboard the ship *Bell Shimada* during Summer 2019, as described in Ramón-Laca et al. (2021) and Shelton et al. (2022b). These were 2.5L samples taken from Niskin bottles deployed at night on a CTD rosette at a maximum of five depths (50, 100, 150, 300 and 500 m) at 186 stations; in addition samples from 3m depth were collected from the ship's saltwater intake line and processed identically to Niskin samples. 347 individual water samples, each representing a unique depth-station combination, are included here. Samples were filtered immediately using a vacuum pump on 47 mm diameter mixed cellulose-ester sterile filters with a 1 μm pore size, and stored at room temperature in Longmire's buffer until DNA extraction.

\[Metadata, as appropriate\] \[note that the 347 samples included here are the intersection of those with hake metabarcoding obs and positive qPCR obs\]

\[OLE: do we have to correct for air hake, or has that already happened w the dataset I've sourced?\]

### qPCR of Reference Species

We used qPCR data for *Merluccius productus* (Pacific Hake), previously published in Shelton et al. 2022b, as a common reference species with which to expand the calibrated metabarcoding proportions into absolute abundance estimates. Hake was present in nearly every sample, and the qPCR data derived from a well-vetted single-species assay (see Ramón-Laca 2021), resulting in a high-confidence reference against which to measure other species' abundances.

```{=tex}
\begin{align}
  z_{ij} &\sim Bernoulli(p_{ij})\\
  logit(p_{ij}) &= \phi_{0}+\phi_{1}\log_{10}C_{ij}\\
  %\\
  Ct_{ij} &\sim Normal(\beta_{0}+\beta_{1}\log_{10}C_{ij},\sigma_{ij}) \qquad  if \: z_{ij} = 1 \\
  \sigma_{ij} &= e^{\theta + \gamma{Ct_{ij}}}
\end{align}
```
where $C_{ij}$ is the concentration of species $j$ in sample $i$, $Ct_{ij}$ is the observed cycle-threshold value in the qPCR assay for species $j$ in sample $i$, etc.

This is a standard logistic-regression model in which the probability of detection is a function of target species concentration, $C$ (Eq. 1 and 2), jointly estimated with the linear model relating the standard curve to field eDNA concentrations (Eq. 3), which itself allows the standard deviation among $Ct$ values to vary with the mean (Eq. 4). Relevant parameters for equations 3 and 4 are estimated using observations of $Ct$ for samples of known concentration (i.e., standards) varying across seven orders of magnitude. See Shelton et al. (2023) and \[earlier dropouts paper\] for similar work.

### Metabarcoding and Sequence Annotation

We amplifed each sample using \[12S Mifish\]

processed via bioinformatics wrapper around cutadapt, dada2

annotation via blast and LCA on highly similar sequences

total reads, total annotations

### Metabarcoding Calibration and Expansion

<!--# Calibrated metabarcoding analysis yielded quantitative estimates of the proportions of species' DNA in environmental samples prior to PCR, following Shelton et al. (2022a).  -->

We generated mock communities of XX fish species derived from total DNA extract from fin or other tissues (SUPPLEMENT). These mocks consisted of equimolar concentrations of total genomic DNA from each species, but following Shaffer et al. (SUBMITTED), we calculated species-specific amplification efficiencies according to the species-specific ddPCR concentrations using the relevant MiFish primer with accompanying SYBR assay. These data reflect the proportions of MiFish-amplifiable DNA for each species in the mixture, rather than proportions of total DNA.

We observed substantial differences in amplification efficiency across species (SUPPLEMENT), and therefore calibration yielded estimates of community composition that different in important ways from the observed uncalibrated proportions. Notably, herring (*Clupea pallasii*) ...

*NOTE -- need to change the math below, to reflect 1) separate estimate of alpha, and 2) dirichlet-multinomial*

```{=tex}
\begin{align}\label{eq:ratios}
  \log\left(\dfrac{Y_i}{Y_R}\right)
  & = \log\left(\dfrac{c_i}{c_R}\right) +
  N_{PCR}\log\left(\dfrac{1+a_i}{1+a_R}\right)
\end{align}
```
We amend the above equation to simplify notation, 1) defining $\log\left(\dfrac{Y_i}{Y_R}\right)$ as $\nu_i$ and 2) $\log\left(\dfrac{1+a_i}{1+a_R}\right)$ as $\alpha_i$, 3) allowing for species-specific process variation, $\epsilon_i$, and 4) separating the initial concentration terms $c$ to yield:

```{=tex}
\begin{align}\label{eq:nu}
  \nu_i & = (\log{c_i} - \log{c_R}) + N_{PCR} \alpha_i + \epsilon_i
\end{align}
```
Sequence-read observations for a given sample, $\boldsymbol{Y}$, are then treated as draws from a multinomial distribution with element proportions $\boldsymbol{\pi}$ and sample size (= sample read depth) $N$. Individual species proportions $\pi_{i}$ are softmax transformations of $\nu_i$ , such that $\sum_{i=1}^I \pi_i = 1$. Where species-specific process variation, $\epsilon_i$, is greater than zero, this is an overdispersed multinomial distribution.

```{=tex}
\begin{align} \label{eq:Ysampling}
  \boldsymbol{Y} & \sim Multinomial(\boldsymbol{\pi},N) \\
  \pi_i &= \dfrac{e^{\nu_i}}{\sum_{i=1}^I e^{\nu_i}} \\
  \epsilon_i &\sim N(0,\tau_i)
\end{align}
```
We preserve estimates of observation variation at the various observation steps, such that they ramify throughout the joint model. For the metabarcoding observations, practical limitations required we sequence only single PCR replicates for most samples; we used \[a handful of replicated samples\] to estimate $\epsilon_i$ for target species, \[and applied that info by applying some fuzz to our predictions...\].

### Spatial-Statistical Analysis

Given estimates of species' DNA concentrations from environmental samples, we generated species-specific spatial models of the distribution of that DNA in space over a XX x YY degree spatial grid of the northeastern Pacific Ocean.

<!--We used the R package sdmTMB to fit the estimated eDNA concentrations for generating species distribution models. Each species was fitted individually while we created and selected the best mesh for approximating spatial Gaussian random fields as part of the model. Owen can add more specifics to this?-->

## Results

### Species Patterns by Latitude and Depth

\[*Figure 1: Estimated species eDNA concentration (A) across 0, 50, 150, 300, and 500m depth samples and known depth distribution of those species from literature (B).*\]

![Figure 1](https://raw.githubusercontent.com/MMARINeDNA/QM-qPCR-joint/refs/heads/main/plots/1.%20Figure_1.jpg)

Our results indicate that eDNA estimated concentration reflects concordantly species distirbution known from literature. We have visuzlised 6 species (for simplicity, for all species see Supplementary Material xxx) eDNA estimated concentration across the US pacific coast in Figure 1A and the known depth habitat range of those species in Figure 1B. 

<!--I am writing a lot of information here just to have it but it can be cut or parts of it moved to discussion. Feel free to move/remove things around-->

**Surface**

Our selected surface species included two clupeoids, northern anchovy and sardines. Northern anchovy (*Engraulis mordax*) is an epipelagic species ranging from Baja, California, to British Columbia, Canada occuring mostly in the photic area above the thermocline. Larvaes and juveniles typically occupy the upper layer of the epipelagic zone 0 - 50m while adults move to surface at night and all the way down to 100m during the day to avoid predators. Their major spawning peaks in July (in accordance with our sampling period) with an epicenter located off Oregon and Washington which is associated with the nutrient rich water near Columbia River plume \cite{litz_2008}. Our eDNA concentration estimates reflected clearly anchovy's signal with high aggregation of concentration and higher occupancy on the surface layers (0 - 50m). Additionally the eDNA abundance estimates signaled the peak abundance matched the spawning location near Columbia River plume suggesting the aggregation of eDNA concentration coming from eggs and larves in surface in addition to adults.

Pacific sardine (*Sardinops sagax*) is an epipelagic specie with very similar distribution to anchoviy in terms of depth ranges with the depth ranges slightly deeper than anchovy and has a preference for warmer waters. The eDNA patters revealed the ecological niche latitudional differences between anchovy and sardines where the former preferes cold surface sea temperatures (SST; 10 ̊–14 ̊C) often located north of California while the latter perefers warmer SST (18–24°C) \cite{nevarez-martinez_2001}.

**Midwater**

Mid water species include mesopelagic species and here we included widow rockfish and lanternfish. Lanternfish is a mesopelagic species occupying the depths of 400 - 700m during the day and 20 - 200m at night \cite{moku2000}. This dial vertical migration (DVM) is due to their feeding strategies as they feed mainly on krill, amphipods, and copepods \cite{moku2000}\cite{suntsov2008} which also does DVM for avoiding predators. Our eDNA concentration estimates signaled that this fish is occuring along the entire water column (studied here) with sparser detection and lower concentrations at the surface. In our eDNA results, there were distinct patches of occurances of latnternfish on the surface, such as near Columbia river, south of Oregon and north of California while in deeper layers the occurence is more ubiquitous. 

Widow rockfish (*Sebastes entomelas*) is typically found at depths ranging from 100 to 250 meters, with juveniles occupying the 50 to 150-meter zone is distributed from northern Baja California to southern Alaska. Widow rockfish exibits schooling behavior, forming dense feeding schools at night preying on euphausiids, salps, sergestids, myctophids, and other small fish which disperse at dawn \cite{Adams_1987}. During summer, widow rockfish are commonly observed at depths between 100 and 150 meters \cite{reynolds_2001}. Our eDNA concentration estimates for Widow rockfish matched previous studies on their abudnance and additionally identified their distribution to be strongly associated with vertical depth within the water column as previously reported \cite{reynolds_2001}.

**Benthic**

The deep water species include benthic dwelling dover sole and dragonfish. Dragonfish (*Tactostoma macropus*) is a mesopelagic to bathypelagic species found in the subarctic and transitional North Pacific typically between 40 to 50 degree latitude \cite{kawaguchi1993}. Adult dragonfish primarily reside in deep waters, occupying depths between 300 and 900 meters. However, their eggs are spawned at depth but rise to the warmer, productive surface layers above the thermocline (0-60m) during summer \cite{kawaguchi1993}, where after hatching feed mainly on copepods and krill \cite{kawaguchi1993}. As they grow, juvenile dragonfish start performing DVM \cite{willis1982}, following their primary prey, krill, to depths around 100 meters \cite{kawaguchi1993}. Thereafter they settle at depths below 300 meters, as their feed type shifts towards myctophids \cite{fisher1983}\cite{kawaguchi1993}. Such bimodal lifestyle activity, especially in the summer, is reflected through the eDNA concentration estimates where the signal from larvae and juveniles is predominantly present near columbia river which has nutrient rich water and is known to be a high productive area in terms of primary consumers and the adults (i.e., 500m deep signal) from mainly two hotspots, near columbia river and northenr california.

Dover sole (*Microstomus pacificus*) is a deep-water flatfish inhabiting depths between 200 and 1200 meters along the Pacific coast, from southern California to the Bering Sea \cite{drazen2007}\cite{brodziak2000}\cite{drazen2012}. This species exhibits ontogenetic migration, with individuals moving progressively deeper as they grow, following the seafloor as they age \cite{vetter1994}\cite{hunter1990}. The eDNA concentration estimates aligned with this pattern, showing high concentrations mostly in deep water samples and in shallow samples when located near the seabed. Additionally two distinct patches of were observed, one in north of Oregon and the other in northern California. <!--They spawn in deep waters (600-1000m) and their eggs are positively boyant which makes them float to surfance but hatch in winter which is not in accordance with our sampling time hence we see no DNA signal at surface (IDK if this is vital info).-->

### Ecological Patterns and Associations with Covariates

\[FIG 2: posterior means vs. covariate values; post-hoc, not built into model\]

-   trends w salinity, temperature, etc. Upwelling index, nutrient / chla

\[FIG 3: map of species overlaps\]

-   species overlaps. Richness peaks at surface at 45 degrees N; attenuates with depth. (Richness also peaks closest to shore, where bathymetric depth is near zero). Is this an artifact of the handful of target species? or effect of the Columbia?

## Discussion

We report patterns of species-specific eDNA concentrations that closely mirror the \[known\] abundance distributions of the species themselves, in three dimensions and over ten degrees of latitude. Our results point the way to a broader use of eDNA for management of commercially important species, such as is required under the Magnuson-Stevens Act in the United States.

### Species Patterns by Latitude and Depth

The eDNA data reflect well-understood latitudinal patterns in most species. For example, the bulk of anchovy being more northerly than sardines, hake being ubiquitous across latitudes, and so on.

<!--# We have developed a quantitative method of simultaneously surveying many wild species, using molecular methods alone. We have reported spatially explicit estimates for 10 species of commercial importance — information of the kind that is required under the Magnuson-Stevens act, which governs commercial fishing in the United States — but the same technique is equally applicable to many questions of ecological management or conservation concern. Below we highlight spatial patterns rising out of the present analysis that illustrate conclusions relevant to management questions. -->

<!--By creating species distribution models and outputs we can better understand species habitat suitabilities. Furthermore coupling it with environmental covariates we can better predict how marine species will shift in response to environmental changes, aiding both fisheries management and biological conservation. This allows for more effective strategies to protect biodiversity while ensuring the sustainability of fisheries resources \cite{liu_2023}.-->

### Species Patterns by Latitude and Depth

The eDNA data reflect well-understood latitudinal patterns in most species. For example, the bulk of anchovy being more northerly than sardines, hake being ubiquitous across latitudes, and so on.

<!-- A bit about abiotic factors influencing the detectabilty of eDNA. If the patterns of fish distribution (cold and warm water species) correspond with their habitat suitability, then eDNA degradation from abiotic / biotic factors can be ballpark estimated to be constant/small accross species over the large (large here is the key word) survey area. If not then it is affecting our observation of certain species (biased towards cold temp species?) -->

Discrete depth sampling has the distinct advantage over other survey methods in so far as it can provide distribution estimates in three dimensions. Widow rockfish, for example, appear almost exclusively in the 50 m depth band, which is unsampled by traditional trawls. This species is of special interest because its commercial harvest has resumed after years of closure, and no existing survey adequately captures its patterns of abundance \[CITE\].

eDNA at depth reflects, to some degree, and attenuated signal from species nearer the surface --- see, for example, the concentration of hake eDNA with depth (Shelton et al. 2022b). However, the widow rockfish pattern suggests this sinking-and-spreading does not dominate signal at lower depths for that species.

<!--# This is likely a function of the overall abundance of species' eDNA and of the species' habitats: highly abundant surface species such as hake and anchovy show attenuated eDNA signals with depth, while less abundant species are more strictly associated with particular sampling depths [SUPPLEMENT]. Abundant surface species' DNA is likely to be a component of marine snow, transported as feces and other waste materials via gravity to lower depths -- rarer species would accordingly make up a smaller fraction of genetic materal raining downward. Hake is more evenly distributed across depths. -->

Eulachon and flatfish eDNA distribution appears to be in conflict with the distribution of harvested adults, most likely reflecting different life-history stages.

<!--low abundance species are very sporadically occurring and hence the maps are not as granular as the other species who are more abundant and the smoothing surface is better. Since those species are at the virge of detection, species occupancy models would yield better results than spatial smoothing. -->

### Ecological Patterns and Associations with Covariates

Our analysis reflects spatial patterns associated with a gradient in temperature -- inseparable from latitude in the present dataset -- but associations with other important ecological covariates are readily apparent. Surface salinity reflects the influence of the Columbia River, for example, and predicts well the presence or absence of anadromous fishes, including eulachon and salmonids. Shifts in the community composition of fishes associated with differences in upwelling index are similarly robust \[signaling and ecological response to a change in physical/oceanographic conditions.\]

<!--Species conservation, overlaps of shared habitat indicate productive and nursing areas thus high importance. -->

### Environmental Smoothing and Sampling Frequency

In general, the scale of environmental sampling limits the resolution with which we can understand spatial variability in a signal. Characterizing a strong, continuous signal -- here, a high concentration of a species' DNA persisting in the environment -- requires fewer sampling points in space than does a more subtle or discrete signal. We therefore expect schooling, high-biomass fish species with broad distributions (sardine, anchovy) to have strong eDNA signals that vary smoothly over space given our sampling design, while more solitary species should be detected more sporadically. Our observations are consistent with these expectations.

The observable eDNA signal is an integrated function of production and loss terms (blurred somewhat by sampling variability, which we address elsewhere; Gold et al. 2023). Qualitatively, if we imagine the rate of eDNA loss (via degradation or sinking) as being fast relative to transport (via surface currents and, to a limited extent, diffusion), an eDNA signal will be detectable only close to a source animal in space and time. The scale over which the eDNA signal is observable will be a function of the ratio of production and loss terms in any given context. We refer to this spatial scale as environmental smoothing of eDNA signal relative to the source animals themselves.

Although eDNA may remain detectable kilometers away from a source (under unidirectional flow; Pont et al; Thalinger et al), it is by far most abundant within meters of the source (Xiong et al. in review). Moreover, the vast majority of eDNA an animal sheds into the environment is lost from surface samples within a period of hours (Brasseale et al in review). The horizontal scale of environmental smoothing in the present dataset is therefore kilometers at most. Because sampling stations are spaced at intervals greater than this \[HOW MANY KM?\], the spatial patterns we report are likely to reflect real differences in abundance rather than artifacts arising from transport.

Vertical smoothing, by contrast, appears minimal -- perhaps at the scale of meters: shallow-water species (sardine, anchovy, herring) occur largely at the surface and 50m depth bins, *Sebastes melanops* occurs almost exclusively at the 50m depth bin (its known habitat; CITE), species found throughout the water column are either diel vertical migrators or species occupying different depths at various life-history stages (Fig 1), and bottom-dwelling fishes are detected primarily at the deepest point sampled on any station.

### Maximizing Information from Existing Surveys

Although we have shown an example of the substantial value of multispecies eDNA analysis for fisheries and other natural-resource management questions, eDNA data alone do not provide information on size- or age structure of the kind important to traditional fisheries management. Moreover, the eDNA sampling itself depends upon fisheries research vessel cruises for access to deep water. Finally fisheries control-rules are expressed in units of mass; converting from DNA concentration to species mass requires data external to the eDNA signal.

Effective management of natural resources at a continental scale requires that we maximize the information content of existing sampling regimes. Here, we follow Shelton et al. (2023) and Liu et al. (in prep?), leveraging the same water samples to provide information on an additional 13 species of commercial importance. Elsewhere, the same set of samples has yielded information on the distribution of marine mammals (Valdivia et al, in revision). We therefore see eDNA as an important datastream that maximizes the information value of existing research cruises, rather than a means of replacing those cruises themselves.

### Observation Model

Unlike single-species observation methods, metabarcoding data yields information on species proportions rather than absolute values. Metabarcoding has no generalizable species-specific detection probability (= sensitivity) because the probability of detection for a given species depends strongly upon an interaction between the other species in the reaction and on the total read-depth for the sample. Where total read-depth is greater, rarer species are more likely to be detected, conditional on their having been sampled and on their amplifiability with the PCR primer being used.

The observation model we use here focuses on species common in the raw dataset, which are those farthest from the lower limits of detection within the present context, and therefore having the highest confidence for consistent detectability.

Our observation model also depends upon detection of hake (our reference species) in both metabarcoding and qPCR datasets for each water sample (\~ 95% ?? of all samples); samples omitted from analysis for lack of hake detections were sporadically distributed in the data, and their omission appears not to have induced substantial bias in our estimates (SUPPLEMENT).

The raw dataset (SUPPLEMENT) contained many more species than we chose to analyze here, nearly all of which were rarer in the data.

## Conclusion

We get a lot of information out of limited number of discrete water samples, about a lot of species simultaneously, with a cost-effective and scaleable technique.

## References

Helping litterature:

\cite{weijerman_2019} Taxonomic composition of fish assemblages varies with depth; Spatial predictions of relative biomass

\cite{zhang_2020a} If you looking for spatial autocorrelation of fish species
