---
title: Quantitative, Multispecies Monitoring at a Continental Scale 
author:
  - Gledis Guri:
      institute: 1
      equal_contributor: "yes"
      correspondence: "yes"
      email: gguri@uw.edu
  - Owen Liu:
      institute: 2
      equal_contributor: "yes"
  - Ryan Kelly:
      institute: 1
  - Megan Shaffer:
      institute: 1
  - Kim Parsons:
      institute: 2
  - Krista Nichols:
      institute: 2
  - Andrew Olaf Shelton:
      institute: 2
institute:
  - 1: University of Washington
  - 2: NOAA fisheries science
mainfont: "Times New Roman"
header-includes:
  - \linespread{1.2}
fontsize: 12pt
geometry: "margin=1in"
format: pdf
---

```{r}
#| echo: false
suppressMessages(library(here))
suppressMessages(library(tidyverse))

```

Target: PNAS, end of 2024

## Change log

+ v3 has a revised introduction by Ole.  References need to be added to the intro.
+ v3 added first paragraph in Results from Ryan. 
+ v3 Ryan rearranged to match PNAS format and condensed methods.
+ v4 Gled refined introduction and added a paragraph on the importance of eDNA being complementary survey method.
+ v4 Gled added findings in abstract + added some information about methods in introduciton to ease transition into results.
+ v4 Gled checked and refined the results + added Figure 4
+ v4 RPK edited/smoothed text

## Abstract

Molecular data from environmental samples can reflect the abundance of species' DNA, an index immediately relevant to natural resources management at a continental scale. These data commonly derive from qPCR or ddPCR assays --- which focus on a single species --- or metabarcoding assays from more general PCR primers that amplify many species simultaneously. Multispecies analyses make efficient use of field samples and laboratory time, and speak to inherently multispecies questions of management and ecological research. However, unlike single-species techniques, metabarcoding by itself can reflect only the proportions of target-species DNA present, not their absolute quantities. Here, we combine qPCR and metabarcoding data derived from the same samples to map quantities of many fish species along the US Pacific coast in three dimensions, demonstrating a technique of practical importance for both management and ecology. We derive spatially explicit estimates of eDNA abundance for 13 common species of commercial and management importance and point the way to quantitative surveys of wild species using molecules alone. Species-distribution maps derived from eDNA capture the broad scale spatial patterns and depth habitats of individual species, and taken together, reflect areas of increased species diversity in regions of ecological significance including the Columbia River plume and the Heceta Bank.

# Intro

<!--*4000 words is standard (6 pages).  8000 words (12 pages max).  Average 4 figures.*-->

Environmental DNA (eDNA, residual genetic material sampled from water, soil, or air) is an increasingly common tool for non-invasively sampling marine and aquatic ecological communities, valuable for estimating species occurrence (\[CITE\]), biodiversity (ie. Richness; Muenzel et al. 2024 PNAS), and genetic structure (Andres et al. 2023 PNAS). However, eDNA has not been widely used for quantitative abundance estimation, upon which many practical management and conservation applications rely (but see \[CITE\] Shelton et al., Guri et al, Stoeckle et al.). While the use of eDNA for quantification is supported both conceptually -- where there is more of a species, more cells are shed in the environment, and thus there is more DNA – and empirically (e.g. there are strong linkages between single-species eDNA measures of in aquaria \[CITE Jo\] Kim Ledger et al. (2024), rivers and streams \[CITE\], estuaries and nearshore habitats \[CITE\], and coastal oceans \[CITE\] [@maes2023], plus [Shelton et al. 2019]), quantitative uses of eDNA observations have significantly lagged behind their use in occurrence and biodiversity applications. 

Quantitative estimates of DNA are more common with species-specific assays (e.g. qPCR and ddPCR) because these approaches directly measure DNA concentration. In contrast, multispecies analysis (metabarcoding) reflects information on many species of interest simultaneously, but at the costs of producing compositional estimates of proportional abundance (Gloor et al; McLaren et al) that may require statistical correction for species-specific biases due to PCR amplification (Shelton et al. 202X). Consequently, metabarcoding data alone cannot yield information about the abundances of DNA from environmental samples (e.g. REFS). 

While trawling and other traditional sampling methods have been the fundamental tools for directly measuring fish abundance and biomass, they are often impractical for species inhabiting inaccessible habitats, occurring at low densities, or being fragile and easily injured during capture. By contrast, eDNA sampling captures information on a broader range of taxa, including those that are difficult or impossible to study with traditional gear. As such, eDNA can meaningfully complement traditional stock assessment methods for commercially important species while at the same time revealing the abundance and distribution of elusive or understudied species.

Here, we develop quantitative estimates for a diverse group of fish species spanning different depth niches across 10 degrees of latitude along the US west coast entirely from eDNA. We integrate single- and multi-species techniques to create three-dimensional distribution maps, leveraging models established in previous studies [@shelton2022, @guri2024a, @shelton2023, @allan2023]. By calibrating metabarcoding data -- correcting raw species observations to their initial proportions using a mock community [@shelton2023] -- and using the absolute eDNA quantity of a reference species (here Pacific Hake (*Merluccius productus*)), we expand initial estimated proportions into absolute estimates of the abundance of each species' eDNA [@guri2024a]. We subsequently smooth the estimated eDNA quantities to generate species distribution maps [@liu2023]. We present results for 13 fish species to illustrate the approach, but note that it immediately generalizes to any ecological community of interest.

<!-- Our study illustrates how eDNA can simultaneously complement distributional and abundance information for well-studied species while revealing new biological details for ecologically important but understudied species.  -->


The eDNA-derived estimates we provide here speak directly to the requirements of natural-resources management at large scales. For example, indices of abundance have a central role in determining stock status for fisheries applications (i.e. stock assessments in the U.S. under the Magnunson-Stevens Act) as well as informing conservation status (e.g., under the U.S. Endangered Species Act). Thus eDNA-derived abundance indices can fill a variety of existing legal requirements for management. While these indices are still in their infancy and will require additional technical developments to ensure their proper usage, they are conceptually and practically little different from abundance estimates derived from more traditional ocean survey method (i.e. net-based or acoustic surveys). Indeed, eDNA methods may provide some notable improvements because single- and multi-species eDNA approaches avoid tradeoffs involved with physically capturing many species simultaneously using a single net. Molecular surveys capture many species with the same "gear" using methods that are easily replicated in space and time. As agencies begin to implement the new National Aquatic eDNA Strategy (REF) -- the first goal of which is to fold eDNA data into federal decisionmaking -- this and other large-scale examples point the way to maximizing the benefit of this information-rich datastream for natural-resources management at a continental scale.

# Results

We find spatial patterns of DNA concentrations reflect the distributions of commercially important coastal-pelagic fish species with remarkable fidelity, in three dimensions and across ten degrees of latitude (Fig. \ref{Fig1}: Northern Anchovy, Pacific Sardine; Supplement Pacific Herring and Jack Mackerel). For other high-biomass species otherwise lacking well-described distributions, we provide the first species distribution models (Fig. \ref{Fig1}, other species). We find that the estimated eDNA concentrations are correlated among species with similar traits and habitat preferences (Fig. \ref{Fig2}). Emergent patterns of species richness and estimated eDNA aggregation correspond with known ecological drivers [NOTE: Gled, need to insert measure of primary productivity, or similar], evidence that eDNA traces can reliably capture ecologically meaningful patterns (Fig. /ref{Fig3}). Particularly for forage fish species -- which are critical high-biomass links in marine food webs, but difficult to survey with traditional methods [CITE Pew Report] -- molecular methods offer a tractable and quantitative index of abundance over large spatial scales.

## Species distribution maps

![Estimated species eDNA concentration (A) across 0, 50, 150, 300, and 500m depth samples and known depth distribution of those species from literature (B)\label{Fig1}](https://raw.githubusercontent.com/MMARINeDNA/QM-qPCR-joint/refs/heads/main/plots/1.%20Figure_1.jpg)

[NOTE -- add latitude to y-axis of upper plots in figure 1]

Below we highlight results for six species representing different depth habitats; all species shown in the supplemental material.

### **Surface**

Northern anchovy (*Engraulis mordax*) inhabits epipelagic waters between Baja California and British Columbia, with larvae and juveniles typically occurring at somewhat shallower depths than the adults [CITE]. The species has a major spawning peak in July (our sampling period), centered on the nutrient-rich water near the Columbia River plume [@litz2008]. Our eDNA estimates mirrored these expected patterns, revealing high concentrations in surface waters at northern latitudes (Fig. \ref{Fig1}; Columbia River outflow at latitude 46.2°N), particularly near the Columbia River outflow. Pacific sardine (*Sardinops sagax*) favors slightly warmer temperatures (SST 18–24°C vs. 10–14°C for the anchovy) and typically inhabits slightly greater depths [@nevarez-martinez2001]. Our estimated eDNA patterns tracked these known differences with sardine eDNA peaking at 41°N (range 40°–45°), while anchovy eDNA peaking at 43.5°N (range 42°–48°; Fig. \ref{Fig4}) indicating how eDNA can capture subtle yet meaningful differences in habitat preferences.

### **Mid-water**

Widow rockfish (*Sebastes entomelas*) are common between 100-250m between northern Baja California and southern Alaska, with juveniles occupying somewhat shallower habitats (50 - 150m) [@reynolds2001] and closer to the coast [@quigley2024]. Individuals form dense feeding schools at night -- preying on euphausiids, salps, and small fishes -- and disperse at dawn [@reynolds2001]. The eDNA concentration estimates for the rockfish are consistent with the shallower end of these expected habitats, with the 50m samples capturing the majority of the signal (Fig. \ref{Fig4}). Similarly, the eDNA concentrations for northern lampfish (*Stenobrachius leucopsarus*) -- an abundant myctophid that migrates from 400-700m up to 20-200m at night [@moku2000] -- are again consistent with the ecology of the species, having notably lower concentrations in surface samples.

### **Benthic**

Among deeper-water fishes, eDNA revealed spatial and ontogenetic patterns for species that are common yet little studied, such as the dragonfish (*Tactostoma macropus*), which occurs roughly between 40°N-50°N (adults at 300-900m, larvae at 0-60m in summer) [@kawaguchi1993]. Juveniles transition to deeper waters as they mature, performing diel vertical migrations following krill to depths around 100 meters [@willis1982; @kawaguchi1993]. The eDNA concentrations appear to reflect this activity (Fig. \ref{Fig4}), with the predominant signal at northern latitudes likely reflecting larvae and juveniles (Fig. \ref{Fig1}), and two deeper concentration hotspots at 300m and 500m separated by about eight degrees of latitude. Also consistent with known habitats are commercially caught flatfish such as Dover sole (*Microstomus pacificus*), which lives at 200m to 1200m along the Pacific coast [@drazen2007;@brodziak2000;@drazen2012]; individuals move progressively deeper as they grow [@vetter1994;@hunter1990]. The eDNA concentration estimates are consistent with this pattern, showing high concentrations mostly in deep water samples (Fig. \ref{Fig4} A) and in shallower samples when located near the seabed (Fig. \ref{Fig1}). 

<!-- Additionally two distinct patches of were observed, one in north of Oregon and the other in northern California (Fig. \ref{Fig1} & Fig. \ref{Fig4} B).  -->

## Species Associations



![Pairwise correlations of eDNA concentrations across species, organized by habitat depths. The upper triangle represents the strength and direction of correlation graphically; the lower triangle gives Pearson's correlation coefficient for each species pair. \label{Fig2}](https://raw.githubusercontent.com/MMARINeDNA/QM-qPCR-joint/refs/heads/main/plots/2.%20Figure_2.jpg)

Species sharing similar habitats have similar spatial distributions of eDNA  (Fig. \ref{Fig2}), particularly with respect to depth: surface, midwater, and deepwater species show distinct patterns of correlation. More subtle differentiation is also apparent, for example chub mackerel (*Scomber japonicus*) and sardine (*Sardinops sagax*) being more common in the more southern (and somewhat warmer) waters (Fig. \ref{Fig4}), while other epipelagic and mesopelagic species have higher eDNA concentrations in more northern waters. [GLED ET AL -- a challenge here is that across the sampled range of latitudes, the mean SST is essentially the same; need better/different info. NOTE ALSO we need to number the figures in the order in which they are referred to.] Concentrations of the remaining surface-dwelling species (northen anchovy, eulachon, jack mackerel, and pacific herring) are modestly correlated with midwater species (Figs.\ref{Fig2} and \ref{Fig4}) consistent with generally colder habitats -- for example,  and further suggesting that eDNA is capturing not only depth differences but also more subtle habitat preferences. Deep-water species are distinct (Figs.\ref{Fig2} and \ref{Fig4}), not strongly correlated with those from other zones. Deeper water tends to feature lower and more diffuse fish eDNA concentrations (Fig. \ref{Fig4} and CITE SHELTON et al), such that any similiarities among species distributions are likely to be more difficult to detect.

<!-- This pattern of correlation and segregation was gradual when species were ordered from the upper to the lower parts of the water column (Fig. \ref{Fig2}), aligning closely with their known habitat preferences. -->

<!-- An evident species for this scenario is Pacific herring which is predominantly aggregated around 48°N but occupies the surface water while the south epipelagic is around 42°N and occupies also the surface water, but the correlation plots indicated that they are different suggesting that eDNA can distinguish habitat differences althuogh the species distribution is wide and species occyping similar depth zones [@Ryan please help to rephrase this sentence]. -->

<!-- Deep-water species form a distinct group with little to no correlation with species from other depth zones apart from *Stenobrachius leucopsarus* and *Tactostoma macropus* which correlated more with the opposite groups (deepwater and midwater respectively). Classifying deepwater species presents challenges due to the vagueness of their definitions. Depending on the study’s purpose, these species may be categorized differently, leading to them interchangeably swapping places between mesopelagic and bathypelagic classifications [@Ryan please help to rephrase this paragraph]. -->

## Species richness 

![A. Species richness. B. Total eDNA concentration across species, an index of total biomass of the species surveyed.\label{Fig4}](https://raw.githubusercontent.com/MMARINeDNA/QM-qPCR-joint/refs/heads/main/plots/3.%20Figure_3.jpg)

Species richness varied substantially and predictably by depth, with the highest richness observed at the surface (Fig. \ref{Fig3} A). In part, this pattern is likely to be an artifact of the focal species in this study -- chosen for their relevance to fisheries management as well as for their abundance in the raw data -- but the richness-depth gradient also reflects the expectation that habitat and [nutrient/oxygen] availability is greatest near the surface [CITE Smith and Brown 2002 "Patterns of diversity, depth range and body size among pelagic fishes along a gradient of depth"; CITE oceanography]. Surface waters also host the eggs, larvae, and juvenile stages of meso- and bathypelagic species -- some of which we detect -- and hence we both expect and observe higher richness in the upper layer of the water column. 

Species richness also corresponds strongly with the influence of Columbia River plume (Fig. \ref{Fig3} A), which extends well beyond the coastline (CITE Hickey and Banas 2003 Estuaries]. The nitrate-rich river water interacts with ocean currents and intensifies coastal upwelling, enhancing both primary and secondary productivity [CITE] and likely influencing the patterns of species richness we observe (Fig. \ref{Fig4} A). The total fish eDNA accumulation (Fig. \ref{Fig3} B) suggests the highest aggregation of forage fish near Heceta bank (44 degrees N; overlapping anchovy and sardine distributions are the two largest contributors to eDNA concentration), a center of abundance for commercial fisheries [CITE], and elsewhere nearshore in areas of coastal upwelling. 

![Species summaries by depth and latitude\label{Fig3}](https://raw.githubusercontent.com/MMARINeDNA/QM-qPCR-joint/refs/heads/main/plots/4.%20Figure_4.jpg)
[CAN we put primary productivity on here, if possible? - RPK]


# Discussion

We report patterns of species-specific eDNA concentrations that closely mirror the \[known\] abundance distributions of the species themselves, at spatial scales relevant to ecology and to natural-resources management. Our results point to a broader use of eDNA for surveying commercially important fish species, such as is required under the Magnuson-Stevens Act in the United States, and more generally for regulatory contexts where large-scale assessments are required.

<!--We have developed a quantitative method of simultaneously surveying many wild species, using molecular methods alone. We have reported spatially explicit estimates for 10 species of commercial importance — information of the kind that is required under the Magnuson-Stevens act, which governs commercial fishing in the United States — but the same technique is equally applicable to many questions of ecological management or conservation concern. Below we highlight spatial patterns rising out of the present analysis that illustrate conclusions relevant to management questions.--> 

<!--By creating species distribution models and outputs we can better understand species habitat suitabilities. Furthermore coupling it with environmental covariates we can better predict how marine species will shift in response to environmental changes, aiding both fisheries management and biological conservation. This allows for more effective strategies to protect biodiversity while ensuring the sustainability of fisheries resources [@liu2023].-->

## Spatial Resolution of eDNA Surveys
<!-- Species Patterns by Latitude and Depth -->

The eDNA data reflect well-understood latitudinal and depth patterns in most species surveyed (Results), and accordingly, the patterns of trace genetic material appear to mirror those of the animals themselves at these spatial scales.

eDNA at depth reflects, to some degree, an attenuated signal from species nearer the surface --- see, for example, the concentration of hake eDNA with depth (Shelton et al. 2022b). However, the widow rockfish pattern suggests this sinking-and-spreading does not dominate signal at lower depths for that species.


<!-- A bit about abiotic factors influencing the detectabilty of eDNA. If the patterns of fish distribution (cold and warm water species) correspond with their habitat suitability, then eDNA degradation from abiotic / biotic factors can be ballpark estimated to be constant/small accross species over the large (large here is the key word) survey area. If not then it is affecting our observation of certain species (biased towards cold temp species?) -->

Discrete depth sampling has the distinct advantage over other survey methods insofar as it can provide distribution estimates in three dimensions. Widow rockfish, for example, appear almost exclusively in the 50 m depth band, which is unsampled by traditional trawls. This species is of special interest because its commercial harvest has resumed after years of closure and no existing survey adequately captures its patterns of abundance \[CITE\].


<!--This is likely a function of the overall abundance of species' eDNA and of the species' habitats: highly abundant surface species such as hake and anchovy show attenuated eDNA signals with depth, while less abundant species are more strictly associated with particular sampling depths [SUPPLEMENT]. Abundant surface species' DNA is likely to be a component of marine snow, transported as feces and other waste materials via gravity to lower depths -- rarer species would accordingly make up a smaller fraction of genetic materal raining downward. Hake is more evenly distributed across depths.--> 

<!-- Eulachon and flatfish eDNA distribution appears to be in conflict with the distribution of harvested adults, most likely reflecting different life-history stages. -->

<!--low abundance species are very sporadically occurring and hence the maps are not as granular as the other species who are more abundant and the smoothing surface is better. Since those species are at the virge of detection, species occupancy models would yield better results than spatial smoothing. -->

<!-- ## Species Associations  -->

<!-- Our analysis reflects spatial patterns associated with a gradient in temperature -- inseparable from latitude in the present dataset -- but associations with other important ecological covariates are readily apparent. Surface salinity reflects the influence of the Columbia River, for example, and predicts well the presence or absence of anadromous fishes, including eulachon and salmonids. Shifts in the community composition of fishes associated with differences in upwelling index are similarly robust \[signaling and ecological response to a change in physical/oceanographic conditions.\] -->

<!--Species conservation, overlaps of shared habitat indicate productive and nursing areas thus high importance. -->

## Environmental Smoothing and Sampling Frequency

In general, the scale of environmental sampling limits the resolution with which we can understand spatial variability in a signal. Characterizing a strong, continuous signal -- here, a high concentration of a species' DNA persisting in the environment -- requires fewer sampling points in space than does a more subtle or discrete signal. We therefore expect schooling, high-biomass fish species with broad distributions (sardine, anchovy) to have strong eDNA signals that vary smoothly over space given our sampling design, while more solitary species should be detected more sporadically. Our observations are consistent with these expectations.

The observable eDNA signal is an integrated function of production and loss terms (blurred somewhat by sampling variability, which we address elsewhere; Gold et al. 2023). Qualitatively, if we imagine the rate of eDNA loss (via degradation or sinking) as being fast relative to transport (via surface currents and, to a limited extent, diffusion), an eDNA signal will be detectable only close to a source animal in space and time. The scale over which the eDNA signal is observable will be a function of the ratio of production and loss terms in any given context. We refer to this spatial scale as environmental smoothing of eDNA signal relative to the source animals themselves.

Although eDNA may remain detectable kilometers away from a source (under unidirectional flow; Pont et al; Thalinger et al), it is by far most abundant within meters of the source (Xiong et al. in review). Moreover, the vast majority of eDNA an animal sheds into the environment is lost from surface samples within a period of hours (Brasseale et al in review). The horizontal scale of environmental smoothing in the present dataset is therefore kilometers at most. Because sampling stations are spaced at intervals greater than this \[HOW MANY KM?\], the spatial patterns we report are likely to reflect real differences in abundance rather than artifacts arising from transport.

Vertical smoothing, by contrast, appears minimal -- perhaps at the scale of meters: shallow-water species (sardine, anchovy, herring) occur largely at the surface and 50m depth bins, *Sebastes melanops* occurs almost exclusively at the 50m depth bin (its known habitat; CITE), species found throughout the water column are either diel vertical migrators or species occupying different depths at various life-history stages (Fig. \ref{Fig1}), and bottom-dwelling fishes are detected primarily at the deepest point sampled on any station.

## Maximizing Information from Existing Surveys

Although we have shown an example of the substantial value of multispecies eDNA analysis for fisheries and other natural-resource management questions, eDNA data alone do not provide information on size- or age structure of the kind important to traditional fisheries management. Moreover, the eDNA sampling itself depends upon fisheries research vessel cruises for access to deep water. Finally fisheries control-rules are expressed in units of mass; converting from DNA concentration to species mass requires data external to the eDNA signal.

Effective management of natural resources at a continental scale requires that we maximize the information content of existing sampling regimes. Here, we follow Shelton et al. (2023) and Liu et al. (in prep?), leveraging the same water samples to provide information on an additional 13 species of commercial importance. Elsewhere, the same set of samples has yielded information on the distribution of marine mammals (Valdivia et al, in revision). We therefore see eDNA as an important datastream that maximizes the information value of existing research cruises, rather than a means of replacing those cruises themselves.

## Observation Model

Unlike single-species observation methods, metabarcoding data yields information on species proportions rather than absolute values. Metabarcoding has no generalizable species-specific detection probability (= sensitivity) because the probability of detection for a given species depends strongly upon an interaction between the other species in the reaction and on the total read-depth for the sample. Where total read-depth is greater, rarer species are more likely to be detected, conditional on their having been sampled and on their amplifiability with the PCR primer being used.

The observation model we use here focuses on species common in the raw dataset, which are those farthest from the lower limits of detection within the present context, and therefore having the highest confidence for consistent detectability.

Our observation model also depends upon detection of hake (our reference species) in both metabarcoding and qPCR datasets for each water sample (\~ 95% ?? of all samples); samples omitted from analysis for lack of hake detections were sporadically distributed in the data, and their omission appears not to have induced substantial bias in our estimates (SUPPLEMENT).

The raw dataset (SUPPLEMENT) contained many more species than we chose to analyze here, nearly all of which were rarer in the data.


# Methods



## Sample Collection, Processing, and Quantitative PCR (qPCR)

Samples were collected in 2.5L Niskin bottles at five depths from stations along the West Coast of the US aboard the ship *Bell Shimada* during Summer 2019, filtered immediately on 47 mm diameter mixed cellulose-ester sterile filters with a 1 $\mu$m pore size, subsequently extracted using a phase-lock phenol-chloroform protocol, and analysed in triplicate using a qPCR assay as described in @ramon-laca2021 and @shelton2022. 
<!-- These were 2.5L samples taken from Niskin bottles deployed at night on a CTD rosette at a maximum of five depths (0, 50, 150, 300 and 500 m) at 186 stations; in addition samples from 3m depth were collected from the ship's saltwater intake line and processed identically to Niskin samples. 347 individual water samples, each representing a unique depth-station combination, are included here. Samples were filtered immediately using a vacuum pump on 47 mm diameter mixed cellulose-ester sterile filters with a 1 $\mu$m pore size, and stored at room temperature in Longmire's buffer until DNA extraction. -->

<!-- DNA extraction of water samples was done using a phase lock phenol-chloroform protocol, as in xxx. Negative controls such as field water and air blanks, and laboratory blanks (extraction blanks) were taken throughout the workflow. The extracted eDNA were analyzed downstream to get estimates of hake DNA concentrations (via quantitative PCR, qPCR) and of fish community composition (via DNA amplicon metabarcoding). -->

<!--\[Metadata, as appropriate\] \[note that the 347 samples included here are the intersection of those with hake metabarcoding obs and positive qPCR obs\]-->

<!--\[OLE: do we have to correct for air hake, or has that already happened w the dataset I've sourced?\]-->

<!-- ## qPCR quantification of Pacific hake -->

<!-- Environmental samples were analysed in triplicate using the qPCR assay described in @ramon-laca2021 as presented in @shelton2022.  -->

<!-- The thermocycler protocol included an initial denaturation step at 95 °C 10 min followed by 40  cycles of 15 s at 95 °C and 1 min at 58 °C. All thermocycler reactions were run in 20 $\mu$L volume consisting of 10 $\mu$L of TaqMan Environmental Master Mix 2.0, 1 $\mu$L of each primer (forward and reverse, 10 nM concentration each), 0.5 $\mu$L of probe (10nM concentration), 3 $\mu$L of dH20 and 2 $\mu$L of DNA template. To monitor quality control, all runs included an internal positive control (IPC) to detect PCR inhibition and a negative template control (NTC) containing water instead of DNA template. Any IPC delay exceeding 0.5 cycles in the NTC was considered inhibition. -->

<!-- To quantify Pacific hake DNA, environmental samples were analyzed alongside standards constructed from a 120 bp synthetic DNA fragment (gBlock; IDT) representing the 12S region of Pacific hake (Merluccius productus), which encompasses the 101 bp 12S qPCR target (sequence information available in Ramón-Laca et al. 2021). The synthetic DNA fragment was diluted to create a series of standards with final concentrations ranging from $10^{0}$ to $10^{5}$ copies/$\mu$L. -->

## Metabarcoding: Environmental Samples and Mock Community

In total, [347] environmental samples were randomized and divided into 7 sequencing libraries for an Illumina MiSeq (v3 600 cycle kit). Total PCR reactions included eDNA samples (n = xxx), PCR blanks (n = xxx), positive controls (n = 8), extraction blanks (n = xxx), fieldwork water and air blank (n = xxx and xxx respectively), were amplified using MiFish-U universal primers [bolded; @miya2015a] with Illumina tails (forward TCG TCG GCA GCG TCA GAT GTG TAT AAG AGA CAG **GCC GGT AAA ACT CGT GCC AGC**; reverse GTC TCG TGG GCT CGG AGA TGT GTA TAA GAG ACAG **CA TAG TGG GGT ATC TAA TCC CAG TTTG**). DNA was amplified in the first PCR reaction (PCR1) in a 20 $\mu$L reaction consisting of: 10 $\mu$L of Phusion Master Mix (2X), 0.4 $\mu$L of the forward primer (10 $\mu$M); 0.4 $\mu$L of the reverse primer (10 $\mu$M), 0.6 $\mu$L of 100% DMSO, 0.5 $\mu$L of recombinant albumin (20 $\mu$g/$\mu$L), 4.4 $\mu$L of nuclease-free water and 2 $\mu$L of DNA template. Reactions were run with the following cycling conditions: 98°C for 30s; 35 cycles of 98°C for 10s, 60°C for 30s, and of 72°C for 3s; 72°C for 10m.

PCR product was cleaned using Ampure Beads (1.2x) and then indexed in a second reaction (12.5 $\mu$L KAPA Hifi HotStart ReadyMix, 1.25 $\mu$L of nucleotide indexes (one per sample, Illumina Corp., IDT for Illumina DNA/RNA UD Index), 5 ul of PCR1 product and 6.25 of nuclease-free water) with cycling conditions of 95°C for 5m; 8 cycles of 98°C for 20s, 56°C for 30s, 72°C for 1m; 72°C for 5m. Products were visualized on a 2% agarose gel and quantified using QuantIt (REF xxx) with Fluoroskan (REF xxx). Indexed products were pooled into libraries for sequencing, and then size- selected to extract only the target fish band using the E-gel Size Select System (REF xxx).

Sequences were de-multiplexed and the adapters removed by Illumina sequencing. Primers were removed with Cutadapt v4.9 [@martin2011]. We then used DADA2 [default parameters, @callahan2016] to denoise sequences, remove chimeras, and generate amplicon sequence variants (ASVs). The sequences were then blasted against the NCBI nucleotide (nt) database (access: August 2024) using BLASTn algorithm with a cut off at 97% identity, assigning the least common ancestor of the top hit, resulting in species, genus, family, or higher rank classifications. Samples with fewer total number of reads than 1000 were removed from downstream analysis as indicator of poor reliability. Bioinformatic pipeline code and relevant data are available [HERE].

To calibrate metabarcoding observations and account for amplification bias [@gold2023; @shelton2023], we generated multiple mock communities comprising a total of 39 fish species. Vouchered DNA extracts or tissues were obtained from either the University of Washington Fish Collection at the Burke Museum or the NOAA Northwest Fisheries Science Center collections. After quantifying the concentration of mitochondrial DNA template in each DNA extract, we constructed a total of eight mock communities (4 even and 4 skewed proportions of mitochondrial DNA) of varying species compositions (see SUPPLEMENTAL METHODS). We selected 13 species of commercial and ecological importance that co-occurred in the environmental samples and in the mock community for downstream analysis: Pacific herring (*Clupea pallasii*), northern anchovy (*Engraulis mordax*), northern smoothtongue (*Leuroglossus stilbius*), Dover sole (*Microstomus pacificus*), Pacific sardine (*Sardinops sagax*), Pacific chub mackerel (*Scomber japonicus*), widow rockfish (*Sebastes entomelas*), northern lampfish (*Stenobrachius leucopsarus*), California barracudina (Tactostoma macropus), plainfin midshipman (*Tarletonbeania crenularis*), eulachon (*Thaleichthys pacificus*), Pacific jack mackerel (*Trachurus symmetricus*), and Pacific hake (*Merluccius productus*). After amplification and sequencing using the protocols outlined above, we used the mock communities to estimate species-specific amplification efficiencies and correct for amplification bias following [@shelton2022].

<!-- [MOVE TO SUPPLEMENT?] We quantified the total genomic DNA in each extract using Qubit HS assay. We then quantified the concentration of the 12S rRNA gene using MarVer1 primers [@valsecchi2020], which perfectly match the template and give unbiased estimates of concentration [SHAFFER et al in review], using ddPCR (Bio-Rad, Inc., QX200 Droplet Digital PCR system). Each tissue of each species was quantified in a 22 $\mu$L reaction consisted of 2 $\mu$L of DNA template from genomic DNA , 11 $\mu$L of ddPCR EvaGreen (Bio-Rad), 0.22 $\mu$L of each forward and reverse primers (10 uM), and 0.56 $\mu$L of nuclease free water. The thermocycler reactions were run in C1000 Touch Thermal Cycler with 96-Deep Well Reaction Module (Bio-Rad) using the PCR program as follows: 2 min at 50°C for enzyme activation, 2 min at 95° C for initial denaturation, and 40 cycles of denaturation for 1 sec at 95°C and primer annealing and elongation for 30 sec min at 60° C, with a ramp rate of 2° C per s and heldat 4° C until droplets were read. Droplets were determined to be positive after drawing a threshold based on NTCs. -->


## Statistical analysis

<!-- We estimated amplification efficiencies of the listed species from mock community metabarcoding run as described in [@shelton2023] using the Stan language as implemented in R (package: Rstan) running 4 MCMC chains with 5000 warmup and 5000 sampling iterations each (see Supplementary math & code).  -->

Having estimated the initial proportions of each species in environmental samples, we then used the absolute quantity of Pacific hake (from qPCR) to expand, for each sample, the proportions into absolute eDNA estimates (copies/$\mu$L) for each species as described in [@guri2024a] and Allan et al. 2023 in a joint model implemented Stan [CITE rstan] (see Supplementary math & code). 

<!-- The posterior of both model predictions were diagnosed using $\hat{R}$ statistics (Gelman and Rubin, 1992) and considered convergence for $\hat{R}$ values less than 1.1 and effective sample size (ESS) greater than 200 for all parameters.  -->

Finally, we smoothed the species-specific DNA concentrations ($C$) at each depth sampled (d = 0, 50, 150, 300 or 500 m) using sdmTMB package [CITE] to generate distribution models for the concentration of eDNA , as:
$$
ln(C_{xyz})=\alpha_d + \omega(s)+\epsilon_d(s)
$$
where $x,y,z$ index longitude, latitude, and depth (respectively), $\alpha_d$ is the spatial intercept for each depth, $\omega(s)$ is the smooth spline as a function of sample depth (in m), and $\epsilon_d(s)$ is a smooth, depth-dependent error term. 

Each species was fitted individually while we created and selected the best mesh for approximating spatial Gaussian random fields as part of the model. [OWEN add detail]

All statistical analysis, figure preparation, and data curation were performed in R [CITE R].

# References

Helping litterature:

\cite{weijerman_2019} Taxonomic composition of fish assemblages varies with depth; Spatial predictions of relative biomass

\cite{zhang_2020a} If you looking for spatial autocorrelation of fish species



# Acknowledgments

We thank Katherine Maslenikov for providing tissues for the mock.
